.github/workflows/checklist.yml

```diff
@@ -0,0 +1,16 @@
+name: Add comment
+
+on:
+- pull_request_target
+
+jobs:
+  comment:
+    name: Checklist
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@master
+    - uses: harupy/comment-on-pr@master
```

Comment:
and here, if we decided to use this action, I think that we should not use `master`

We should pin a certain commit.

This will make sure that things will not break.

At the same time, if somehow `harupy/comment-on-pr` repo is accidentally compromised, we will not automatically run the latest code.

At least we run a known commit ID.

----------

There is the still possibility for the whole repo to be compromises so that a trusted commit `abc123` from 3 years ago is replaced with a new commit with the same `abc123` hash , but with malicious code.

Reply:
Thank you — that is a really good point!  This is advice I should probably apply in some other repositories also.